src/components/VideoPlayer.vue

```diff
@@ -587,6 +587,11 @@ export default {
                 const rate = this.getPreferenceNumber("rate", 1);
                 videoEl.playbackRate = rate;
                 videoEl.defaultPlaybackRate = rate;
+
+                const autoDisplayCaptions = this.getPreferenceBoolean("autoDisplayCaptions", false);
+                if (autoDisplayCaptions) {
+                    this.$player.setTextTrackVisibility(!this.$player.isTextTrackVisible());
```

Comment:
Shouldn't this just be `setTextTrackVisibility(true)`?

Reply:
Yes that's right, sorry for the confusion.